orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -453,14 +464,19 @@ def _get_forward_graph(self, input_names, dynamic_axes, *inputs, **kwargs):
             sample_inputs_copy.append(model_input.data if isinstance(model_input, torch.Tensor) else model_input)
         sample_inputs_copy = copy.deepcopy(tuple(sample_inputs_copy))
 
+        sample_kwargs_copy = {}
+        for name, model_input in kwargs.items():
+            sample_kwargs_copy[name] = model_input.data if isinstance(model_input, torch.Tensor) else model_input
```

Comment:
This actually refers to the same data as the original ```model_input```. I think we need to ```copy.deepcopy``` the model_input here.